x-pack/plugins/observability_solution/apm/public/embeddable/alerting/alerting_latency_chart/chart.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import LatencyChart from '../../../components/alerting/ui_components/alert_details_app_section/latency_chart';
+import { ENVIRONMENT_ALL } from '../../../../common/environment_filter_values';
+import { useAlertingProps } from '../use_alerting_props';
+import { TimeRangeCallout } from '../time_range_callout';
+import type { EmbeddableApmAlertingLatencyVizProps } from '../types';
+import { ServiceNameCallout } from '../service_name_callout';
+
+export function APMAlertingLatencyChart({
+  rule,
+  alert,
+  serviceName,
+  environment = ENVIRONMENT_ALL.value,
+  transactionType,
+  transactionName,
+  rangeFrom = 'now-15m',
+  rangeTo = 'now',
+  latencyThresholdInMicroseconds,
+  kuery = '',
+  filters,
+}: EmbeddableApmAlertingLatencyVizProps & {
+  latencyThresholdInMicroseconds?: number;
```

Comment:
Is this necessary? The `latencyThresholdInMicroseconds` prop is already defined in the EmbeddableApmAlertingLatencyVizProps  https://github.com/elastic/kibana/pull/179078/files?file-filters%5B%5D=.json&file-filters%5B%5D=.jsonc&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&owned-by%5B%5D=cauemarcondes&show-viewed-files=true#diff-416512c8a5dad65e9ea2f85e0c22344dd755df952d699db6839cd898aaa0814cR27